### ISSUE TITLE
Couldn't install the OTX by using pypi or pip install -e

### ISSUE THREAD

**Issue by sungmanc** [open-edge-platform/training_extensions#2039]
**Describe the bug**

When I followed the installation guideline, I got the error below,
![image](https://user-images.githubusercontent.com/80735344/233298073-337c0720-6502-4494-8f2a-329e7f8052d0.png)

**Steps to Reproduce**
Following the document for Install OpenVINO Training Extensions for users
![image](https://user-images.githubusercontent.com/80735344/233298678-573f9487-f29d-4c30-a441-ffdd235f7a58.png)

**Environment:**

- OS: Ubuntu 20.04
- Framework version: Pytorch 1.13
- Python version: 3.10

**Comment by goodsong81** [comment]
`python -m pip install --upgrade pip` will help you.
* On pip==22.3.1: same issue as reported
* On pip==23.1: no issues

@sungmanc Would you try with the latest pip?
cc: @jaegukhyun

**Comment by goodsong81** [comment]
Additionally, I'm replacing the `otxdeploy`, which is our custom source distribution, by `mmdeploy` wheel from the official PyPI. This will resolve above issue from otx==1.2.1. (if it's not available on the PyPI, please try `releases/1.2.1` branch)

**Comment by sungmanc** [comment]
When I updated the PyPI version, there is no issues in installation. However, the installation still not working. Now I got an error like the below, (Maybe MMCV installation is missed)
![image](https://user-images.githubusercontent.com/80735344/233876080-a8125ee6-590a-4471-b8e4-3c2f4c81f741.png)

For the unit test, also there is a problem.
![image](https://user-images.githubusercontent.com/80735344/233876105-942db79d-ba74-49e3-9333-190d2b0813d9.png)
